src/config-file-schema.test.ts

```diff
@@ -20,6 +20,12 @@ const schema: JSONSchemaType<TestSchema> = {
   additionalProperties: false,
 };
 
+const ajv = new Ajv({
+  schemas: [schema],
```

Comment:
For my curiosity, is this step required? I didn't have it before so I just want to understand when it's needed and when it isn't

Reply:
I had it before so I included it here, but removed it now to see if it'll work, and it does, so I guess the answer is that this step is not required! :) I'll update the PR right now.